src/transformStateWithClones.js

```diff
@@ -65,7 +65,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const stateResult = [];
```

Comment:
```suggestion
  const stateClons = [];
```

Reply:
Igor is right. `Result` is not the best name for the variable